tests/s390x_tests/lib/main_common.pm

```diff
@@ -0,0 +1,1909 @@
+        if (check_var('VIDEOMODE', 'text') && check_var('BACKEND', 'ipmi')) {
+            loadtest "installation/disable_grub_graphics";
+        }
+
+        if (check_var("UPGRADE", "LOW_SPACE")) {
+            loadtest "installation/disk_space_release";
+        }
+        if (ssh_key_import) {
+            loadtest "installation/ssh_key_setup";
+        }
+        loadtest "installation/start_install";
+    }
+    return 1 if get_var('EXIT_AFTER_START_INSTALL');
+    load_common_installation_steps_tests;
+    if (is_sles4sap()) {
+        if (check_var('SLES4SAP_MODE', 'sles4sap_wizard')) {
+            loadtest "installation/sles4sap_wizard";
+            if (get_var("TREX")) {
+                loadtest "installation/sles4sap_wizard_trex";
+            }
+            if (get_var("NW")) {
+                loadtest "installation/sles4sap_wizard_nw";
+            }
+            loadtest "installation/sles4sap_wizard_swpm";
+        }
+    }
+}
+
+sub load_console_server_tests {
+    if (check_var('BACKEND', 'qemu') && !is_jeos) {
+        # The NFS test expects the IP to be 10.0.2.15
+        loadtest "console/yast2_nfs_server";
+    }
+    loadtest "console/http_srv";
+    loadtest "console/dns_srv";
+    loadtest "console/postgresql_server" unless (is_leap('<15.0'));
+    # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+    if (is_sle && sle_version_at_least('12-SP1')) {    # shibboleth-sp not available on SLES 12 GA
+        loadtest "console/shibboleth";
+    }
+    if (!is_staging && (is_opensuse || get_var('ADDONS', '') =~ /wsm/ || get_var('SCC_ADDONS', '') =~ /wsm/)) {
+        # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+        loadtest "console/pcre" if is_sle;
+        # TODO test on SLE https://progress.opensuse.org/issues/31972
+        loadtest "console/mysql_odbc" if is_opensuse;
+        if (is_leap('<15.0') || is_sle('<15')) {
+            loadtest "console/php5";
+            loadtest "console/php5_mysql";
+            loadtest "console/php5_postgresql96";
+        }
+        loadtest "console/php7";
+        loadtest "console/php7_mysql";
+        loadtest "console/php7_postgresql96";
+    }
+    # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+    loadtest "console/apache_ssl" if is_sle;
+    # TODO test on openSUSE https://progress.opensuse.org/issues/31972
+    loadtest "console/apache_nss" if is_sle;
+}
+
+sub load_consoletests {
+    return unless consolestep_is_applicable();
+    if (get_var("ADDONS", "") =~ /rt/) {
+        loadtest "rt/kmp_modules";
+    }
+
+    loadtest "console/consoletest_setup";
+
+    if (get_var('IBM_TESTS')) {
+        # prepare tarballs for the testcase
+        # the path below should be reworked to be universal for any distribution, now it's for openQA deployed on opensuse 
+        my $tcs_path = "/var/lib/openqa/share/tests/sle/data/s390x/";        
```

Comment:
I think those pathes are not correct anymore, no?

Reply:
The path `my $tcs_path = "/var/lib/openqa/share/tests/sle/data/s390x/";`
is exactly what we have in our openQA server which is based on openSuSE
so in our environment it's correct path
```
osus2:/var/lib/openqa/share/tests/sle/data/s390x # ls
TOOL_s390_CHCHP  TOOL_s390_ZIOMON  TOOL_s390_fcp  TOOL_s390_hyptop  TOOL_s390_qethqoat  TOOL_s390_vmcp  lib

osus2:/var/lib/openqa/share/tests/sle/data/s390x # cat /etc/os-release
NAME="openSUSE Leap"
VERSION="42.3"
...

osus2:/var/lib/openqa/share/tests/sle/data/s390x # ls -la /var/lib/openqa/share/tests
total 12
drwxr-xr-x  3 geekotest root 4096 Oct  2 03:10 .
drwxr-xr-x  4 root      root 4096 Aug  6 08:07 ..
drwxr-xr-x 10 geekotest www  4096 Oct  2 03:10 opensuse
lrwxrwxrwx  1 root      root    9 Jul 17 07:15 sle -> opensuse/

```